admin/packaging.py

```diff
@@ -864,7 +880,23 @@ def omnibus_package_builder(
             BuildPackage(
                 package_type=distribution.package_type(),
                 destination_path=destination_path,
-                source_paths={flocker_node_path: FilePath("/usr/sbin")},
+                source_paths={
+                    flocker_node_path: FilePath("/usr/sbin"),
+                    # Fedora/CentOS firewall configuration
+                    package_files.child('flocker-control-api.firewalld.xml'):
```

Comment:
Wouldn't we only want applicable files for each OS to be installed, rather than installing everything? Not nice to dump random crap in `/etc` if it's for different OS.